lib/vdsm/tool/config_lvm_filter.py

```diff
@@ -218,9 +225,20 @@ def _print_filter_warning():
     """)
 
 
+def _print_devices_info(vgs):
+    print("""\
+Configuring LVM system.devices.
+Devices for following VGs will be imported:
+    """)
+    print(f" {', '.join(vgs)}")
+    print()
+
+
 def _print_summary(mounts, current_filter, wanted_filter, advice_wwids):
     _print_mounts(mounts)
-    _print_recommended_filter(wanted_filter)
+
+    if wanted_filter:
+        _print_recommended_filter(wanted_filter)
```

Comment:
Devices info should be printed here, in the same way we print the wanted filter.